ui/src/components/AppHeader.js

```diff
@@ -1,56 +1,69 @@
 import React from 'react';
 import { makeStyles } from '@material-ui/core/styles';
 
-import AppBar from '@material-ui/core/AppBar';
 import IconButton from '@material-ui/core/IconButton';
-import Toolbar from '@material-ui/core/Toolbar';
-import Typography from '@material-ui/core/Typography';
 import MenuIcon from '@material-ui/icons/Menu';
 
 const useStyles = makeStyles(theme => ({
-  appBar: ({ drawerWidth }) => ({
+  appBar: {
     position: 'sticky',
-    ...(drawerWidth && {
-      [theme.breakpoints.up('sm')]: {
-        width: `calc(100% - ${drawerWidth}px)`,
-        marginLeft: drawerWidth,
-      },
-    }),
-  }),
+    [theme.breakpoints.up('sm')]: {
+      display: 'none',
+    },
+    padding: '8px',
+    display: 'flex',
+    justifyContent: 'space-between',
+    alignItems: 'center',
+  },
   menuButton: ({ drawerWidth }) => ({
     marginRight: theme.spacing(2),
     ...(drawerWidth && {
       [theme.breakpoints.up('sm')]: {
         display: 'none',
       },
     }),
+    height: '48px',
   }),
-  title: {
-    flexGrow: 1,
+  logo: {
+    height: '60px',
+    width: '200px',
+    display: 'flex',
+    alignItems: 'center',
+    '& a': {
+      display: 'flex',
+      alignItems: 'center',
+    },
+  },
+  logoImage: {
+    transform: 'scale(0.85)',
   },
 }));
 
 function AppHeader({ handleDrawerToggle, children, drawerWidth }) {
   const classes = useStyles({ drawerWidth });
 
   return (
-    <AppBar className={classes.appBar}>
-      <Toolbar>
-        <IconButton
-          edge="start"
-          className={classes.menuButton}
-          color="inherit"
-          aria-label="menu"
-          onClick={handleDrawerToggle}
-        >
-          <MenuIcon />
-        </IconButton>
-        <Typography variant="h6" className={classes.title} noWrap>
-          Agoric Core HQ
-        </Typography>
-        {children}
-      </Toolbar>
-    </AppBar>
+    <div className={classes.appBar}>
+      <div className={classes.logo}>
+        <a href="https://agoric.com">
+          <img
+            className={classes.logoImage}
+            src="https://agoric.com/wp-content/themes/agoric_2021_theme/assets/img/logo.svg"
```

Comment:
This introduces an external constraint on agoric.com's URL layout. Do we document that somewhere?

Reply:
Moved this to a shared constant to make this clearer